config.json

```diff
@@ -184,7 +184,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "icon": "features-oop",
+      "title": "Modern",
+      "content": "VB.NET is a modern, fast-evolving language."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Cross-platform",
+      "content": "VB.NET runs on almost any platform and chipset."
+    },
+    {
+      "icon": "features-functional",
+      "title": "Multi-paradigm",
+      "content": "VB.NET is primarily an object-oriented language, but also has lots of functional features."
+    },
+    {
+      "icon": "features-lazy",
+      "title": "General purpose",
+      "content": "VB.NET is used for a wide variety of workloads, like websites, console applications, and even games."
```

Comment:
I don't think VB.net is used for games a lot. Maybe replace that with GUI applications or something like that? Note that the max length is 100 characters.

Reply:
Yes, Angelika pointed that out to me a couple of days ago.